src/Microsoft.Health.SqlServer/Features/Schema/SchemaInitializer.cs

```diff
@@ -232,6 +227,12 @@ private async Task<bool> CanInitializeAsync(CancellationToken cancellationToken)
                             return false;
                         }
                     }
+                    else
+                    {
+                        _logger.LogWarning("Database creation is not enabled.");
+
+                        throw new InvalidOperationException(Resources.NewDatabaseCannotBeCreated);
```

Comment:
Is this the right type of exception to be throwing here?

Reply:
As discussed with @rbans96 and @brandonpollett, I removed the exception that is thrown and instead return false in 253f2af. An exception should be thrown once database creation in PR environments is enabled. I created a story for this [here](https://microsofthealth.visualstudio.com/Health/_workitems/edit/77870).